openslides_backend/models/models.py

```diff
@@ -2,8 +2,9 @@
 
 from openslides_backend.models import fields
 from openslides_backend.models.base import Model
+from openslides_backend.shared.patterns import Collection
```

Comment:
Where does this come from/is this needed?

Reply:
Ok, it seems to be a relict in `generate_models.py`. Can you remove it from the template there?